bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWbb/Template/proc_card.dat

```diff
@@ -0,0 +1,3 @@
+import model NMSSMHET_UFO --modelname
+generate g g > h03, (h03 > h01 h02, h01 > b b~, (h02 > W+ W-, W+ > j j, W- > j j))
```

Comment:
h02 seems not in the model, it should be h2

Reply:
Hi @menglu21 , I actually see it in the model, if you look at particles.py you'll find this block:
```
h02 = Particle(pdg_code = 35,
                       name = 'h02',
                       antiname = 'h02',
                       spin = 1,
                       color = 1,
                       mass = Param.MH02,
                       width = Param.WH02,
                       texname = 'h02',
                       antitexname = 'h02',
                       charge = 0,
                       GhostNumber = 0,
                       Y = 0)
```
Also, I tried to generate some events and the h02 particle (pdg_Id = 35) is correctly included in the LHE file

Reply:
interesting, because i tried locally and MG asked me to use h2. so please ignore my comment if it goes correctly in your sides.

Reply:
@ammitra can you confirm that you're also able to produce a h02 resonance with your cards and model? If yes I guess we can merge the PR if there are no further comments

Reply:
Hi @mlizzo I can confirm that I'm able to produce the h02 resonance. I tested it out with a gridpack for the `(1300, 200)` mass point and it completed successfully, with a similar `1.99e+19 +- 1.286e+24 pb` xsec to yours.